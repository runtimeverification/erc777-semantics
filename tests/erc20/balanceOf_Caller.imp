test.preamble();

// Testing balanceOf(Owner) when Owner is the caller.

test.balanceOf(caller);
