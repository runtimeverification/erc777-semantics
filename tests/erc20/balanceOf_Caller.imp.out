Balance of 7 is 70
